ephios/core/models/users.py

```diff
@@ -123,13 +124,15 @@ def get_short_name(self):
         return self.first_name
 
     @property
-    def age(self):
+    def age(self) -> Optional[int]:
+        if self.date_of_birth is None:
+            return None
```

Comment:
I think this could go sideways in a few places where calculations are done on the age, as we expected it to be always  there. Did you check to see where this is used? (also thinking about the youthwarden plugin)

Reply:
Yeah also looks fine...